test/js/components/projects/detail.test.jsx

```diff
@@ -314,11 +318,43 @@ describe('<ProjectDetail />', () => {
 
       expect(heading).toBeVisible();
 
-      await fireEvent.click(getByTitle('Close'));
+      fireEvent.click(getByTitle('Close'));
```

Comment:
It often works either way, but `fireEvent` returns a promise so it's probably best-practice to always use `await` with it.

Reply:
oops, was listening to vs code warnings saying `await` would have no affect.

Reply:
Interesting -- I've never seen that warning, and I'm not sure how VSC would know whether it has an effect 🤔

Reply:
@duggiemitchell You're right -- I was getting confused with the Vue version, where `fireEvent` does return a promise. These aren't needed here 👍